mobile/README.md

```diff
@@ -0,0 +1,43 @@
+# CommIT Smart
+
+[commitsmart.com](https://commitsmart.com)
+
+[info@commitsmart.com](mailto:info@commitsmart.com)
+
+## Mobile Assignment
+
+The goal of the assignment is to get a picture of your programming skills and coding style.
+
+You can choose any programming language that you're comfortable with.
+
+Feel free to use 3rd party frameworks and libraries when needed, but try to choose lightweight ones because we'd also like to see how you think and code.
+
+Don't stress if you can't get every part done -- spend a maximum of 3 hours on this assignment. Feel free to add comments or README notes to help us understand your thinking and decisions.
+
+Submit your code through `github.com` by creating a **private** repository and sharing it with the `csabaujvari` github account.
+
+---
+
+## Assignment
+
+1. Create a Flutter application that displays a list of products and allows users to add, edit, and delete them.
+1. Use a responsive design to make the application look good on both iOS and Android devices.
+1. Implement a custom business logic that involves calculating the total price of the products in the cart and applying discounts based on the total price.
+1. Use a local database (e.g. sqflite) to store the products, and use the provider package to manage the state of the application.
+1. Use Git for version control and GitHub for code sharing.
+1. Create a detailed document that explains the architecture of the application, the technologies used, and the reasoning behind the design decisions made, especially the business logic for calculating the total price and applying discounts.
+1. Share your code repository with `csabaujvari` github account.
+
+---
+
+## Evaluation Criteria
+
+1. The application should be functional and correctly perform CRUD operations on the database.
+1. Proper use of Flutter, responsive design, and local database.
+1. Proper use of the provider package to manage the state of the application.
```

Comment:
I thought here there should be a bulletpoint that states something like "Have the different parts or layers of the app its own clear purpose" -> maybe my wording here needs to be improved, but I want to courage them to have a good architecture behind the whole application

Reply:
> Divide the application into different parts or layers, each with its own clear purpose, such as presentation layer, business logic layer, data access layer, and networking layer.

What do you think?

Reply:
looks good 🦄